package.json

```
@@ -22,6 +22,9 @@
     "@mui/x-charts": "^8.14.0",
```

Biome (2.1.2)

[error] The key @mui/x-charts was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)